lib/syscall_shim/syscall_entertab.lds.S

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2024, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <uk/arch/limits.h> /* for __PAGE_SIZE */
+#include <uk/plat/common/common.lds.h> /* for UK_SEGMENT_DATA */
+
+SECTIONS
+{
+	. = ALIGN(__PAGE_SIZE);
```

Comment:
Should that be `__PAGE_SIZE` or would 8 suffice? (e.g. `uklibparam`, `earlytab`). Same question for `exittab`.